src/styles/SearchForm.css

```diff
@@ -0,0 +1,49 @@
+body {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+.wrapper {
+  width: 100%;
+  max-width: 31.25rem;
+  margin: 6rem auto;
+}
+
+.label {
+  font-size: 0.625rem;
+  font-weight: 400;
+  text-transform: uppercase;
+  letter-spacing: +1.3px;
+  margin-bottom: 1rem;
+}
```

Comment:
Are these `.wrapper` and `.label` classes used anywhere? If not they should be removed